docs/query-builder.txt

```diff
@@ -9,588 +9,1095 @@ Query Builder
+- :ref:`Retrieve distinct values <laravel-query-builder-distinct>`
+- :ref:`Aggregations <laravel-query-builder-aggregations>`
+
+.. _laravel-query-builder-where-example:
+
+Where Method Example
+~~~~~~~~~~~~~~~~~~~~
+
+The following example shows how to use the ``where()`` query
+builder method to retrieve documents from the ``movies`` collection
+that contain an ``imdb.votes`` field value of ``350``. Click the
+:guilabel:`{+code-output-label+}` button to see the results returned
+by the query:
+
+.. io-code-block::
+   :copyable: true
+
+   .. input:: /includes/query-builder/MovieController.php
+      :language: php
+      :dedent:
+      :start-after: begin query where
+      :end-before: end query where
+
+   .. output::
+      :language: json
+      :visible: false
+
+      [
+         { "title": "Cosmos",
+           "year": 1980,
+           "runtime": 60,
+           "imdb": {
+             "rating": 9.3,
+             "votes": 17174,
+             "id": 81846
+           },
+           "plot": "Astronomer Carl Sagan leads us on an engaging guided tour of the various elements and cosmological theories of the universe.",
+           ...
+         },
+         { "title": "The Shawshank Redemption",
+           "year": 1994,
+           "runtime": 142,
+           "imdb": {
+             "rating": 9.3,
+             "votes": 1521105,
+             "id": 111161
+           },
+           "plot": "Two imprisoned men bond over a number of years, finding solace and eventual redemption through acts of common decency.",
+           ...
+         },
+         { "title": "The Real Miyagi",
+           "year": 2015,
+           "runtime": 90,
+           "imdb": {
+             "rating": 9.3,
+             "votes": 41,
+             "id": 2313306
+           },
+           "plot": "The life of the greatest karate master of a generation.",
+            ...
+         }
+      ]
+
+.. _laravel-query-builder-logical-operations:
+
+Logical Conditional Operations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The examples in this section show the query builder syntax you
+can use to perform the following logical conditional operations:
+
+- :ref:`Logical OR to match one or more conditions <laravel-query-builder-logical-or>`
+- :ref:`Logical AND to match all conditions <laravel-query-builder-logical-and>`
+- :ref:`Logical NOT to match the negation of the condition <laravel-query-builder-logical-not>`
+- :ref:`Nested logical operator groups <laravel-query-builder-logical-nested>`
+
+.. _laravel-query-builder-logical-or:
+
+Logical OR Example
+^^^^^^^^^^^^^^^^^^
+
+The following example shows how to chain the ``orWhere()``
+query builder method to retrieve documents from the
+``movies`` collection that either match the ``year``
+value of ``1955`` or match the title "``Back to the Future``":
+
+.. literalinclude:: /includes/query-builder/MovieController.php
+   :language: php
+   :dedent:
+   :start-after: begin query orWhere
+   :end-before: end query orWhere
+
```

Comment:
Good question! 
Yes, I purposely excluded it. In earlier drafts, I had started including the output for everything. However, I consciously decided to exclude output of the examples in which the output wouldn't contribute to building understanding since they would either have to know details about the dataset and/or the result is very similar to the description of the query.